scripts/microcleanfs.py

```diff
@@ -29,22 +32,30 @@
 from ampy.pyboard import Pyboard, PyboardError
 from docopt import docopt
 
-
 def main(args: List[str]) -> None:
     opts = docopt(__doc__, argv=args)
     port = opts['PORT']
+    force = opts['--force']
 
     print('Connecting to {}'.format(port), file=sys.stderr)
     board = Pyboard(port)
     files = Files(board)
+    exclude_files = ['boot.py']     # Specifying subdirectories DOES NOT work as they will
```

Comment:
These comments are formatted not in accordance with PEP 8. Could you please put them above the line with `exclude_files = ...` and use the indent level of the current block? PyCharm has a PEP 8 inspection that underlines any PEP 8 problems.

Reply:
Understood.

Reply:
It's a really minor issue, I just want to keep the source code clean :) Thanks for understanding!

Reply:
No worries.  I just wonder why the inspections aren't showing in my PyCharm instance.  Something else to look at...